modules/spo-inspecting-seccomp-profiles.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * security/security_profiles_operator/spo-troubleshooting.adoc
+
+:_content-type: PROCEDURE
+[id="spo-inspecting-seccomp-profiles_{context}"]
+= Inspecting seccomp profiles
+
+Corrupted `seccomp` profiles can disrupt your workloads. Ensure that the user cannot abuse the system by:
+
+* Not creating that user on the actual node.
+* Restricting the user ID to only the `openshift-security-profiles`.
```

Comment:
This comes from the upsteram documentation but even I have trouble parsing what is the information telling me. I guess the first point is to avoid users ssh-ing to the nodes and touching the profiles directly?
What's the second point about?

@saschagrunert do you remember?

Reply:
We're talking about the rootless user `2000` here, which should be unprivileged on the system.

Reply:
Ah!
In that case I'm not sure this is required documentation for OCP, because there is no (supported) way of creating users on the nodes (I guess you can create an oneshot systemd unit that would call useradd or write stuff to /etc/passwd, but...)

I think we should leave the third point.

In general, wearing my compliance hat, maybe we should recommend to install auditd rules that would watch the directory, but..later :-)